tests/testaddress.nim

```diff
@@ -8,7 +8,7 @@
 import unittest2
 import ../chronos
 
-when defined(nimHasUsed): {.used.}
+{.used.}
```

Comment:
I wish to leave here old variant which is more safe and maybe it helps our users to use `chronos` with older versions of Nim.

Reply:
`used` exists - we support 1.2+ only, you won't be able to compile chronos with earlier versions